requirements.txt

```diff
@@ -0,0 +1,10 @@
+setuptools~=46.1.3
+requests~=2.23.0
+python-dotenv~=0.13.0
+pandas~=1.0.3
+tdameritrade~=0.1.0
```

Comment:
this shouldn't be needed, right?

Reply:
We're still using requests in the session.py handler. I had tried to swap it out for request-html, hoping to remove ChromeDriver, but couldn't get it to work. Pandas is used for the DF functions. 

Dotenv is probably something I loaded for my development, but since pipenv loads the .env files this may not be needed.